### PR TITLE
Add test for native pipe in `commented_code_linter()`

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: tests/testthat/test-commented_code_linter.R [hunk 1]
```diff
@@ -84,13 +84,15 @@ test_that("commented_code_linter can detect operators in comments and lint corre
 
   test_ops <- c(
     "+", "=", "==", "!=", "<=", ">=", "<-", "<<-", "<", ">", "->",
-    "->>", "%%", "/", "^", "*", "**", "|", "||", "&", "&&", "%>%",
-    "%anything%"
+    "->>", "%%", "%||%", "/", "^", "*", "**", "|", "||", "&", "&&",
```
Comment (IndrajeetPatil) [inline tests/testthat/test-commented_code_linter.R:87]: From rlang

File: tests/testthat/test-commented_code_linter.R [hunk 2]
```diff
@@ -84,13 +84,15 @@ test_that("commented_code_linter can detect operators in comments and lint corre
...
+    "%>%", "%<>%", "%T>%", "%anything%"
```
Comment (IndrajeetPatil) [inline tests/testthat/test-commented_code_linter.R:88]: From magrittr

**Comment by AshesITR** [comment]
Why would `commented_code_linter()` need more tests regarding the pipe?

**Comment by IndrajeetPatil** [comment]
> Why would commented_code_linter() need more tests regarding the pipe?

Because, currently, we are only testing the magrittr pipe, and not the native pipe.

**Comment by AshesITR** [comment]
But from a code analysis point-of-view, all the SPECIALs are equivalent, i.e. `%anything%` and `%>%` are essentially the same thing for the parser.
I'm fine with adding a `|>` test just-in-case, but the others seem excessive.

**Review by AshesITR — APPROVED** [review]
Thanks 😊

**Pull request merged**